presto-docs/src/main/sphinx/connector/vertica.rst

```diff
@@ -0,0 +1,71 @@
+=================
+Vertica Connector
+=================
+
+The Vertica connector allows querying and creating tables in an
+external `Vertica <https://www.vertica.com/>`_ database. This can be used to join data between
+different systems like Vertica and Hive, or between different
+Vertica instances.
+
+Configuration
+-------------
+
+To configure the Vertica connector, create a catalog properties file
+in ``etc/catalog`` named, for example, ``vertica.properties``, to
+mount the Vertica connector as the ``vertica`` catalog.
+Create the file with the following contents, replacing the
+connection properties as appropriate for your setup:
+
+.. code-block:: none
+
+    connector.name=vertica
+    connection-url=jdbc:vertica://example.net:5433
+    connection-user=root
+    connection-password=secret
+
+Multiple Vertica Servers
```

Comment:
This section is a copy paste of other connectos and basically should be written once somewhere in more generic space about defining catalogs. Let's remove it.